x-pack/plugins/cases/server/client/cases/find.ts

```diff
@@ -104,3 +96,11 @@ export const find = async (
     });
   }
 };
+
+const asArray = <T>(field?: T | T[] | null): T[] | undefined => {
```

Comment:
Is there any chance that we have this utility function available somewhere else?

Reply:
hmm I don't see anything that is generic. I'll move this to our `common/utils` though 👍